recipes/dgl/build.sh

```diff
@@ -32,7 +49,7 @@ cmake -DUSE_CUDA=${USE_CUDA} \
   -DEXTERNAL_METIS_LIB_PATH=${BUILD_PREFIX}/lib \
   -DUSE_LIBXSMM=ON \
   -DUSE_OPENMP=ON \
-  -DCUDA_ARCH_NAME=Maxwell \
+  -DCUDA_ARCH_NAME=All \
```

Comment:
non-cuda build are failing tests now. Maybe add CUDA_ARCH_NAME to CUDA_CMAKE_OPTIONS so it is only used when CUDA is enabled.

Reply:
Yes that was a good surprise to see. Good idea!